tutorials/en_us/PLUGINS/FAST_CHUNK_PREGENERATOR.md

```diff
@@ -50,3 +50,69 @@ FCP has some useful commands that allow you to pause, resume and cancel generati
 You can have a queue of generation tasks by running the command in [Getting Started](FAST_CHUNK_PREGENERATOR.md#getting_started) multiple times. 
 
 To view the pending generation task run the command `/fcp pending`
+
+## Config
+
+
+```
+# Max millis the generation thread should take per tick.
```

Comment:
what's a `millis`
Please be clear and change all instances to `milliseconds` or `ms`

Reply:
i mean these are from the default config. i dont want to change it exactly